README.md

```diff
@@ -216,6 +217,21 @@ link_design top
 write_db reg1.db
 ```
 
+#### Abstract Lef Support
+OpenROAD also contains an abstract lef writer that can take your current design
+and emit an abstract lef representing the external pins of your design and metal
+obstructions.
+
+
+```tcl
+read reg1.db
+write_abstract_lef reg1_abstract.lef
+```
+
+##### Limitations of the Abstract Lef Writer
+Currently the writer will place an obstruction over the entire block area on any
+metal layer if there is any object on that metal layer.
+
```

Comment:
Is OpenROAD able to use these LEFs with that limitation?

Reply:
Maybe it's phrased poorly, but the obstruction should only cover the macro. Basically saying you can't route though a macro. 

With the caveat that if the macro doesn't use a metal layer routing though the macro on that metal layer is allowed.

With that said do you think there's a sentence that makes the docs more clear, or did you have something else in mind?

Reply:
The phrasing "Currently" implies this will change.

My comment was more with respect to can tritonroute find and connect the pins if they are covered by OBS? Most of the macros I deal with have cutouts around the pins, so the OBS doesn't cover the pins and some basic keepout around the pins.

Reply:
It is always permissible to touch a pin even if it is covered by an obs.  You can't go outside the pin shape at all though so it either needs to be on an edge or a via will have to drop from above.

Reply:
I'm really only targeting edge pins with this initial version. I suspect others will be able to extend it to add more complex internal pin dynamics.

For now though this should work with macros with non internal pins. Any objections @arlpetergadfort?

Reply:
@QuantamHD no, sounds like TritonRoute would be okay it, so then it don't see any issues.